jstz_node/src/main.rs

```diff
@@ -46,14 +51,28 @@ async fn main() -> io::Result<()> {
 
     let rollup_client = Data::new(RollupClient::new(rollup_endpoint));
 
+    let (broadcaster, handler, cancel_read_file) =
+        LogService::init(args.kernel_file_path);
+
```

Comment:
Not necessarily in this PR, but it would be nice to come up w/ a trait for services